mon-pix/app/styles/pages/_skill-review.scss

```
@@ -137,6 +137,10 @@
 
 .skill-review-retry {
 
+  &__message {
+    margin-bottom: $spacing-s;
```

```suggestion
    margin: 1em 0;
```

Comme pour les épreuves ?